integration-test-core/src/main/kotlin/integration/core/H2Setting.kt

```diff
@@ -6,6 +6,7 @@ public interface H2Setting<DATABASE : Database> : Setting<DATABASE> {
     override val dbms: Dbms get() = Dbms.H2
     override val createSql: String
         get() =
+            // language=h2
```

Comment:
These comments let IDEA syntax-highlight everything inside the string literal - please let me know if I should remove them.